app/services/folio/types.rb

```diff
@@ -0,0 +1,79 @@
+module Folio
+  class Types
+    class << self
+      delegate :policies, :circulation_rules, :criteria, to: :instance
+    end
+
+    def self.instance
+      @instance ||= new
+    end
```

Comment:
is this just the singleton pattern? I thought you could do `include Singleton` or something

Reply:
You could, except if you want to, say, test it with different parameters.